modules/administration/pages/monitoring.adoc

```diff
@@ -579,15 +605,72 @@ This section demonstrates how to generate client/server certificates for Prometh
 
 .Procedure: Creating server/client TLS certificate
 
-. On the {productname} Server, at the command prompt, run following command:
+. At the command prompt of the {productname} container host, as root, run the following commands:
+
++
+
+--
+
+.. To generate certificate files, run the following command.
+
++
+
+Ensure that the [systemitem]``set-cname`` parameter is the fully qualified domain name (FQDN) of your Salt client.
+You can use the the [systemitem]``set-cname`` parameter multiple times if you require multiple aliases:
```

Comment:
Actually, I'm not fully sure, but I think that's just optional.
Better ask @witekest